native/main.ts

```diff
@@ -163,6 +164,18 @@ ipcMain.handle(RendererMessage.OPEN_URL, (_event, url) => {
   openExternalLink(url);
 });
 
+ipcMain.handle(RendererMessage.USERS_GIT_DEFAULT_BRANCH, () => {
+  try {
+    return childProcess
+      .execSync("git config --global --get init.defaultBranch", {
+        encoding: "utf-8",
+      })
+      .trim();
+  } catch {
+    return undefined;
```

Comment:
We should probably at least log a warning if this fails.

Reply:
Are you referring to any specific type of log? I mean, you are not suggesting to show a notification error for this, is that right?

Reply:
Showing a notification is too intrusive. The logging is mainly for debugging purposes. Since this is the main process and not the renderer process using `console.warn` will print something to `stderr`.